apps/condo/domains/notification/schema/_internalSendNotificationNewMobileAppVersionService.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Generated by `createservice notification._internalSendNotificationNewMobileAppVersionService --type mutations`
+ */
+
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getSchemaCtx } = require('@open-condo/keystone/schema')
+
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const access = require('@condo/domains/notification/access/_internalSendNotificationNewMobileAppVersionService')
+const { DEVICE_PLATFORM_IOS, DEVICE_PLATFORM_ANDROID, APP_MASTER_KEY, APP_RESIDENT_KEY } = require('@condo/domains/notification/constants/constants')
+const { MessageBatch, RemoteClient } = require('@condo/domains/notification/utils/serverSchema')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { RESIDENT, STAFF } = require('@condo/domains/user/constants/common')
+
+const _internalSendNotificationNewMobileAppVersionService = new GQLCustomSchema('_internalSendNotificationNewMobileAppVersionService', {
+    types: [
+        {
+            access: true,
+            type: `enum App { ${APP_RESIDENT_KEY}, ${APP_MASTER_KEY} }`,
+        },
+        {
+            access: true,
+            type: `enum Platform { ${DEVICE_PLATFORM_ANDROID}, ${DEVICE_PLATFORM_IOS} }`,
+        },
+        {
+            access: true,
+            type: 'input _internalSendNotificationNewMobileAppVersionInput { dv: Int!, sender: JSON! platform: Platform!, app: App!, buildVersion: String!, title: String, body: String, organizationIds: [ID!] }',
+        },
+        {
+            access: true,
+            type: 'type _internalSendNotificationNewMobileAppVersionOutput { messageBatchId: ID! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.can_internalSendNotificationNewMobileAppVersion,
+            schema: '_internalSendNotificationNewMobileAppVersion(data: _internalSendNotificationNewMobileAppVersionInput!): _internalSendNotificationNewMobileAppVersionOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+
+                const { data: { dv, sender, platform, app, buildVersion, title, body, organizationIds } } = args
+
+                const { keystone } = await getSchemaCtx('MessageBatch')
+                const adminContext = await keystone.createContext({ skipAccessControl: true })
+
+                const userType = app === RESIDENT ? RESIDENT : STAFF
+                let remoteClientKeys
+                //The case when a push is sent to all suitable devices, regardless of organization
+                if (isEmpty(organizationIds)) {
+                    remoteClientKeys = await loadListByChunks({
```

Comment:
Is there any estimations on list size? Thousands / millions rows? Maybe it can fail later with OOM error